tests/fXmlRpc/Parser/AbstractParserTest.php

```diff
@@ -697,4 +697,17 @@ public function testEmptyStructMember()
         $this->assertSame(array('FIRST' => ''), $this->parser->parse($string, $isFault));
         $this->assertFalse($isFault);
     }
+
+    public function testThrowExceptionWhenIsString()
```

Comment:
Yes, but I must do this in each class, that test case cannot be in AbstractParserTest.